Example-iOS/Source/Main.storyboard

```diff
@@ -31,16 +31,16 @@
                             <outlet property="delegate" destination="oFD-NX-sZZ" id="el9-mJ-yKI"/>
                         </connections>
                     </tableView>
-                    <navigationItem key="navigationItem" title="Examples" prompt="Discover Ways to Use Rive" id="D8Z-my-oJi"/>
+                    <navigationItem key="navigationItem" title=" " prompt="Discover Ways to Use Rive" id="D8Z-my-oJi"/>
```

Comment:
I decided the title looked weird so close to the section headers. So I switched to a prompt instead of a title. I left a space in the title so that it keeps the size of the title bar